_posts/2018-07-10-react-native-wdio.md

```diff
@@ -14,20 +14,20 @@ authors:
 
 We want to automate the validation of our user experiences in order to consistently maintain a high level of quality.
 
-The Mobile App team I work on utilizes native code for parts of our application, yet, most of the user interfaces are written using React Native and JavaScript. Given that most of the UI is written in JavaScript, we wanted to use a similar tech stack for automating the validation of our user experiences (also known as function tests) on Android and iOS platforms.
+The Mobile App team I work on utilizes native code for parts of our application, yet, most of the user interfaces are written using React Native and JavaScript. Given that most of the UI is written in JavaScript, we wanted to use a similar tech stack for automating the validation of our user experiences (also known as functional tests) on Android and iOS platforms.
```

Comment:
`similar tech stack for automating the validation`
similar tech stack for the automation and validation